app/controllers/spree/admin/images_controller_decorator.rb

```diff
@@ -0,0 +1,10 @@
+Spree::Admin::ImagesController.class_eval do
```

Comment:
do we need controller decorator and model decorator together?

Reply:
No I will remove the controller and the product_details_page_spec.rb